### DISCUSSION TITLE
How to minify files in Production mode ?

### DISCUSSION THREAD

**Discussion by KareemDa** [nitrojs/nitro#367] [General]
Hello, I've deployed a new Nuxt 3 app to vercel. Everything is going right **EXCEPT:**

output files are not minified and I didn't figure out how to minify them (maybe I should set production mode somewhere ? )

output: 
![image](https://user-images.githubusercontent.com/42810389/180931017-af7f24be-ba97-4930-b640-df533d25f28b.png)

**Comment by KareemDa** [comment]
Note: I've tried to add `NODE_ENV=production` to `env` files. Same result

**Comment by danielroe** [comment]
What makes you think that isn't minified?

**Comment by KareemDa** [comment]
@danielroe 
Thank you for your response. Actually, Lighthouse gave me a warning that my files need to be minified. 

After a deep lock. It turns out that chrome is prettying the output so I was thinking that It's not minified. (you can notice in the image that the file is formatted but no lines on the left side of the file. means that they're on the same line)

However, I've figured CSS inside ejs are not minified: check the image.

![image](https://user-images.githubusercontent.com/42810389/181062720-7bf8550c-2667-4972-a13b-74769bfda145.png)

**Reply by danielroe** [reply]
It really depends what is producing that CSS. CSS within strings isn't likely something that rollup can minify without writing a custom plugin.

**Reply by KareemDa** [reply]
That's right. I appreciate your time and help!
